test-project/Assets/Generated/Source/improbable/gdk/tests/blittabletypes/BlittableComponentMonoBehaviourCommandHandlers.cs

```diff
@@ -114,6 +114,20 @@ public CommandRequestSender(Entity entity, EntityManager entityManager, ILogDisp
                     return request.RequestId;
                 }
 
+                public long SendFirstCommandRequest(EntityId entityId, global::Improbable.Gdk.Tests.BlittableTypes.FirstCommandRequest payload,
+                    Action<FirstCommand.ReceivedResponse> callback, uint? timeoutMillis = null, bool allowShortCircuiting = false)
+                {
+                    if (!VerifyNotDisposed())
```

Comment:
Rename the function to get rid of this double-negative. e.g. `IsValid`

Reply:
This isn't my function and that drive by will take longer. Can do that in another PR.

Reply:
(actually I look into how many places this function is. It might just be a string replace)